### PR TITLE
Add a few more nice to have pre-commit hooks

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Review by twangboy — CHANGES_REQUESTED** [review]

**Review by dmurphy18 — DISMISSED** [review]
Just the comment on the support URL

File: SUPPORT.rst [hunk 1]
```diff
@@ -30,4 +30,3 @@ guidelines for filing bug reports:
 **SaltStack Support** - If you need dedicated, prioritized support, please
 consider a SaltStack Support package that fits your needs:
 `<http://www.saltstack.com/support>`_
```
[the remark this answers is not in the document]
Comment (s0undt3ch) [reply SUPPORT.rst:32]: That URL is a 404. And should be fixed on a PR specifically addressing that.
This is just automated fixes provided by the pre-commit hooks.

File: pkg/macos/pkg-resources/conclusion.rtf [hunk 2]
```diff
@@ -5,29 +5,29 @@
 \vieww12000\viewh14900\viewkind0
 \deftab720
 \pard\pardeftab720\sa321\partightenfactor0
-{\field{\*\fldinst{HYPERLINK "http://saltstack.com/"}}{\fldrslt 
+{\field{\*\fldinst{HYPERLINK "http://saltstack.com/"}}{\fldrslt
 \f0\fs24 \cf2 \ul \ulc2 Salt}}
-\f0\fs24  has been installed into 
+\f0\fs24  has been installed into
```
[the remark this answers is not in the document]
Comment (s0undt3ch) [reply pkg/macos/pkg-resources/conclusion.rtf]: Should be fixed now

File: pkg/macos/pkg-resources/welcome.rtf [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
+\cf0 This package will install Salt on your Mac. Salt installs into
 \f1 /opt/salt
 \f0 .\
 Sample configuration files (
 \f1 master.dist
-\f0  and 
+\f0  and
 \f1 minion.dist
-\f0 ) will be installed to 
+\f0 ) will be installed to
```
[the remark this answers is not in the document]
Comment (s0undt3ch) [reply pkg/macos/pkg-resources/welcome.rtf]: Should be fixed now

**Review by dmurphy18 — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Comment by s0undt3ch** [comment]
PR approved and passing. Merging.

**Pull request merged**